core/src/main/scala/org/apache/spark/executor/Executor.scala

```diff
@@ -130,6 +130,11 @@ private[spark] class Executor(
   private val urlClassLoader = createClassLoader()
   private val replClassLoader = addReplClassLoaderIfNeeded(urlClassLoader)
 
+  // Load executor plugins
+  Thread.currentThread().setContextClassLoader(replClassLoader)
+  Utils.loadExtensions(classOf[ExecutorPlugin], conf.get(EXECUTOR_PLUGINS), conf)
```

Comment:
not a big deal but any reason we did this in between setting up the classloaders, perhaps move down a few lines just to keep all the classloader stuff together.